server/src/lib/gist/__tests__/index.ts

```diff
@@ -0,0 +1,126 @@
+import { Post } from "@lib/models/Post"
```

Comment:
oh cool, I wonder who added the testing framework.  🙏

Reply:
😊